examples/blender/blender.py

```diff
@@ -92,19 +94,26 @@ async def worker(ctx: WorkContext, tasks):
         budget=10.0,
         timeout=timeout,
         subnet_tag=subnet_tag,
+        driver=driver,
+        network=network,
         event_consumer=log_summary(log_event_repr),
     ) as executor:
 
+        sys.stderr.write(
+            f"Using payment driver {TEXT_COLOR_YELLOW}{executor.driver}{TEXT_COLOR_DEFAULT} "
+            f"and network {TEXT_COLOR_YELLOW}{executor.network}{TEXT_COLOR_DEFAULT}\n"
+        )
+
         async for task in executor.submit(worker, [Task(data=frame) for frame in frames]):
             print(
-                f"{utils.TEXT_COLOR_CYAN}"
+                f"{TEXT_COLOR_CYAN}"
                 f"Task computed: {task}, result: {task.result}, time: {task.running_time}"
-                f"{utils.TEXT_COLOR_DEFAULT}"
+                f"{TEXT_COLOR_DEFAULT}"
             )
 
 
 if __name__ == "__main__":
-    parser = utils.build_parser("Render blender scene")
+    parser = build_parser("Render blender scene")
```

Comment:
actually, if we're changing this line, we could improve it a bit ;)

```suggestion
    parser = build_parser("Render a Blender scene")
```

Reply:
all right, all right